Update _index.md (#31312)
DataDog/documentation: content/en/ddsql_editor/_index.md

{{< callout url="https://www.datadoghq.com/product-preview/logs-metrics-support-in-ddsql-editor/" >}}

> Querying Logs and Metrics through DDSQL is in Preview. Use this form to request access.

Reviewer: ```suggestion
Querying Logs, Metrics, Spans, and RUM through DDSQL is in Preview. Use this form to request access.
```

> Spans and RUM as data sources are also in Preview. Mention in the same form if you want access or even data sources not listed in the use cases section.

Reviewer: ```suggestion
If you want access to Spans, RUM, or other data sources not listed in the use cases section, mention them in the access request form.
```

## Explore your telemetry

> <div class="alert alert-warning">Querying Logs and Metrics through DDSQL is in Preview. Use this <a href="https://www.datadoghq.com/product-preview/logs-metrics-support-in-ddsql-editor/">form</a> to request access.

Reviewer: ```suggestion
<div class="alert alert-warning">Querying Logs, Metrics, Spans, and RUM through DDSQL is in Preview. Use this <a href="https://www.datadoghq.com/product-preview/logs-metrics-support-in-ddsql-editor/">form</a> to request access.
```

> Spans and RUM as data sources are also in Preview. Mention in the same form if you want access or even data sources not listed in the use cases section.

Reviewer: ```suggestion
If you want access to Spans, RUM, or other data sources not listed in the use cases section, mention them in the access request form.
```

{{< img src="ddsql_editor/data-tab.png" alt="The data tab showing table information for aws.ec2_instance" style="width:70%;" >}}

> For data sources like Logs, use the query builder to help you generate table functions.

Reviewer: ```suggestion
For data sources such as Logs, use the query builder to generate table functions.
```
